cmd/gindoid/dataset.go

```diff
@@ -52,7 +52,7 @@ func createRegisteredDataset(job *RegistrationJob) error {
 	} else if storeURL, err := url.Parse(conf.Storage.StoreURL); err == nil {
 		storeURL.Path = path.Join(job.Metadata.Identifier.ID, zipfname)
 		archiveURL = storeURL.String()
-		job.Metadata.Sizes = []string{humanize.IBytes(uint64(zipsize))}
+		job.Metadata.Sizes = &[]string{humanize.IBytes(uint64(zipsize))}
```

Comment:
Maybe I missed something and everything required is already provided via the libgin library, but this commit addresses "Sizes" and "Subjects", but not the mentioned "FundingReferences".

Reply:
Yes, the FundingReferences are handled by libgin and populated in the `NewDataCiteFromYAML()` method like here
https://github.com/G-Node/gin-doi/blob/bb262c1b1b718c0bf59effdda1520129f1f47af7/cmd/gindoid/web.go#L224